### DISCUSSION TITLE
2024/07/14/google_colab_mount_gdriver_and_config_PATH/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#25] [General]
# 2024/07/14/google_colab_mount_gdriver_and_config_PATH/

如题。

https://wz.anoms.top/2024/07/14/google_colab_mount_gdriver_and_config_PATH/

**Comment by Hazelandinstant** [comment]
哇！根据这篇文章我可以无痛跟练嘛？！
有什么问题我就直接留言🤩

**Reply by cyclinbox** [reply]
当然可以的~
有问题欢迎随时交流
（虽然感觉你的语气不是很友善 :( 如果是我做错了什么，向你表示抱歉……）

**Reply by Hazelandinstant** [reply]
啊啊啊啊。。。。可是我觉得自己语气很友善呀（思考脸/非常认真）
是否我的语气过于活泼？？？

**Reply by cyclinbox** [reply]
没有没有……可能是我误解了你的语气😂
PS：一周过去了，话说你有继续探索colab嘛，不知道探索的过程是否顺利？

**Reply by Hazelandinstant** [reply]
没，最近在忙一些自己的事情。

**Reply by cyclinbox** [reply]
加油加油
